.github/workflows/on_push_main.yml

```
@@ -207,9 +236,20 @@ jobs:
       WHEEL_ARTIFACT_NAME: linux-x64-wheel
     secrets: inherit
 
+  test-wheel-macos-arm64:
+    needs: [checks, build-wheel-macos-arm64]
+    name: "macos-arm64: Test Wheels"
+    uses: ./.github/workflows/reusable_test_wheels.yml
+    with:
+      CONCURRENCY: nightly-macos-arm64
```

Copy-paste remainders, perhaps?
```suggestion
      CONCURRENCY: push-macos-arm64-${{ github.ref_name }}-${{ inputs.CONCURRENCY }}
```